db/sql/sql_test.go

```diff
@@ -448,6 +523,9 @@ var _ = Describe("Sql", func() {
 				Expect(resSql).To(Equal(expectedSql))
 				Expect(param).To(Equal(expectedParam))
 			})
+			It("should create disjunction for more than one parameter with like queries", func() {
+				checkLikeDisjunctionAndConjunction("123","42","__or__","%123%","%42%")
```

Comment:
`go fmt` should make a space after each comma.

Reply:
Tip: set an after-save-hook in your editor to run `gofmt`, or even better, `goimports`.

Reply:
thanks @cierpuchaw